2025-05-03T01:59:32.718089+02:00 victim-ubuntu login[66549]: PAM unable to dlopen(pam_lastlog.so): /usr/lib/security/pam_lastlog.so: cannot open shared object file: No such file or directory
2025-05-03T01:59:32.718559+02:00 victim-ubuntu login[66550]: PAM unable to dlopen(pam_lastlog.so): /usr/lib/security/pam_lastlog.so: cannot open shared object file: No such file or directory
2025-05-03T01:59:32.718613+02:00 victim-ubuntu login[66549]: PAM adding faulty module: pam_lastlog.so
2025-05-03T01:59:32.718659+02:00 victim-ubuntu login[66550]: PAM adding faulty module: pam_lastlog.so
2025-05-03T01:59:33.145715+02:00 victim-ubuntu login[66549]: pam_unix(login:auth): check pass; user unknown
2025-05-03T01:59:33.145841+02:00 victim-ubuntu login[66549]: pam_unix(login:auth): authentication failure; logname=.telnet uid=0 euid=0 tty=/dev/pts/3 ruser= rhost=10.10.10.4 
2025-05-03T01:59:33.145896+02:00 victim-ubuntu login[66550]: pam_unix(login:auth): check pass; user unknown
2025-05-03T01:59:33.145928+02:00 victim-ubuntu login[66550]: pam_unix(login:auth): authentication failure; logname=.telnet uid=0 euid=0 tty=/dev/pts/2 ruser= rhost=10.10.10.4 
2025-05-03T01:59:36.483746+02:00 victim-ubuntu login[66549]: FAILED LOGIN (1) on '/dev/pts/3' from '10.10.10.4' FOR 'UNKNOWN', Authentication failure
2025-05-03T01:59:36.484153+02:00 victim-ubuntu login[66550]: FAILED LOGIN (1) on '/dev/pts/2' from '10.10.10.4' FOR 'UNKNOWN', Authentication failure
2025-05-03T01:59:37.501121+02:00 victim-ubuntu login[66773]: PAM unable to dlopen(pam_lastlog.so): /usr/lib/security/pam_lastlog.so: cannot open shared object file: No such file or directory
2025-05-03T01:59:37.501550+02:00 victim-ubuntu login[66772]: PAM unable to dlopen(pam_lastlog.so): /usr/lib/security/pam_lastlog.so: cannot open shared object file: No such file or directory
2025-05-03T01:59:37.501654+02:00 victim-ubuntu login[66772]: PAM adding faulty module: pam_lastlog.so
2025-05-03T01:59:37.501748+02:00 victim-ubuntu login[66773]: PAM adding faulty module: pam_lastlog.so
2025-05-03T01:59:37.589395+02:00 victim-ubuntu login[66773]: pam_unix(login:auth): check pass; user unknown
2025-05-03T01:59:37.589664+02:00 victim-ubuntu login[66773]: pam_unix(login:auth): authentication failure; logname=.telnet uid=0 euid=0 tty=/dev/pts/3 ruser= rhost=10.10.10.4 
2025-05-03T01:59:37.589723+02:00 victim-ubuntu login[66772]: pam_unix(login:auth): check pass; user unknown
2025-05-03T01:59:37.589761+02:00 victim-ubuntu login[66772]: pam_unix(login:auth): authentication failure; logname=.telnet uid=0 euid=0 tty=/dev/pts/2 ruser= rhost=10.10.10.4 
2025-05-03T01:59:44.207522+02:00 victim-ubuntu login[66772]: FAILED LOGIN (1) on '/dev/pts/2' from '10.10.10.4' FOR 'UNKNOWN', Authentication failure
2025-05-03T01:59:45.255364+02:00 victim-ubuntu login[66775]: PAM unable to dlopen(pam_lastlog.so): /usr/lib/security/pam_lastlog.so: cannot open shared object file: No such file or directory
2025-05-03T01:59:45.255490+02:00 victim-ubuntu login[66775]: PAM adding faulty module: pam_lastlog.so
2025-05-03T01:59:45.706686+02:00 victim-ubuntu login[66775]: pam_unix(login:auth): check pass; user unknown
2025-05-03T01:59:45.706781+02:00 victim-ubuntu login[66775]: pam_unix(login:auth): authentication failure; logname=.telnet uid=0 euid=0 tty=/dev/pts/2 ruser= rhost=10.10.10.4 
2025-05-03T01:59:58.642785+02:00 victim-ubuntu login[66991]: PAM unable to dlopen(pam_lastlog.so): /usr/lib/security/pam_lastlog.so: cannot open shared object file: No such file or directory
2025-05-03T01:59:58.642893+02:00 victim-ubuntu login[66991]: PAM adding faulty module: pam_lastlog.so
2025-05-03T01:59:59.225760+02:00 victim-ubuntu login[66991]: pam_unix(login:auth): check pass; user unknown
2025-05-03T01:59:59.225861+02:00 victim-ubuntu login[66991]: pam_unix(login:auth): authentication failure; logname=.telnet uid=0 euid=0 tty=/dev/pts/3 ruser= rhost=10.10.10.4 
2025-05-03T02:00:05.476538+02:00 victim-ubuntu login[66994]: PAM unable to dlopen(pam_lastlog.so): /usr/lib/security/pam_lastlog.so: cannot open shared object file: No such file or directory
2025-05-03T02:00:05.476691+02:00 victim-ubuntu login[66994]: PAM adding faulty module: pam_lastlog.so
2025-05-03T02:00:19.884172+02:00 victim-ubuntu login[67210]: PAM unable to dlopen(pam_lastlog.so): /usr/lib/security/pam_lastlog.so: cannot open shared object file: No such file or directory
2025-05-03T02:00:19.884347+02:00 victim-ubuntu login[67210]: PAM adding faulty module: pam_lastlog.so
2025-05-03T02:00:46.847042+02:00 victim-ubuntu login[67430]: PAM unable to dlopen(pam_lastlog.so): /usr/lib/security/pam_lastlog.so: cannot open shared object file: No such file or directory
2025-05-03T02:00:46.847134+02:00 victim-ubuntu login[67430]: PAM adding faulty module: pam_lastlog.so
2025-05-03T02:01:01.354733+02:00 victim-ubuntu login[67432]: PAM unable to dlopen(pam_lastlog.so): /usr/lib/security/pam_lastlog.so: cannot open shared object file: No such file or directory
2025-05-03T02:01:01.354932+02:00 victim-ubuntu login[67432]: PAM adding faulty module: pam_lastlog.so
2025-05-03T02:01:27.101309+02:00 victim-ubuntu login[67434]: PAM unable to dlopen(pam_lastlog.so): /usr/lib/security/pam_lastlog.so: cannot open shared object file: No such file or directory
2025-05-03T02:01:27.101439+02:00 victim-ubuntu login[67434]: PAM adding faulty module: pam_lastlog.so
2025-05-03T02:01:40.508818+02:00 victim-ubuntu login[67434]: pam_unix(login:auth): check pass; user unknown
2025-05-03T02:01:40.508900+02:00 victim-ubuntu login[67434]: pam_unix(login:auth): authentication failure; logname=.telnet uid=0 euid=0 tty=/dev/pts/3 ruser= rhost=10.10.10.4 
2025-05-03T02:01:43.704963+02:00 victim-ubuntu login[67434]: FAILED LOGIN (1) on '/dev/pts/3' from '10.10.10.4' FOR 'UNKNOWN', Authentication failure
